azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkit/lib/appservice/service/impl/FunctionAppDeploymentSlot.java

```diff
@@ -75,7 +75,7 @@ protected FunctionDeploymentSlot remote() {
 
     private FunctionApp getParentFunctionApp() {
         return StringUtils.isNotEmpty(entity.getId()) ?
-                azureClient.functionApps().getById(ResourceId.fromString(entity().getId()).parent().id()) :
+                azureClient.functionApps().getById(ResourceId.fromString(entity.getId()).parent().id()) :
```

Comment:
if `functionApp` is passed as constructor parameter, this remote call can be saved. right?